linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -1025,12 +1021,10 @@ private final class Analyzer(config: CommonPhaseConfig,
 
         if (!isNativeJSClass) {
           for (clazz <- superClass) {
-            clazz.jsNativeLoadSpec match {
-              case None =>
-                staticDependencies += clazz.className
-              case Some(loadSpec) =>
-                addLoadSpec(externalDependencies, loadSpec)
-            }
+            if (clazz.isAnyClass)
```

Comment:
Could we invert this an use `isNativeJSClass`? `isAnyClass` isn't really meaningful IMHO.

Reply:
Done.